src/types/controlPlaneConfig.ts

```diff
@@ -8,6 +8,17 @@ export type DestinationDefinition = {
   Config: Record<string, unknown>;
 };
 
+export type AccountDefinition = {
+  Name: string;
+  Config: Record<string, unknown>;
```

Comment:
These will be the list of field available in workspace config for transformer you need authenticationType definitely to understand what to do with account options and secrets

  name: string;
  type: string;
  category: string;
  authenticationType: string;
  config: Record<string, unknown>;
  createdAt: Date;
  updatedAt: Date;

Reply:
Added authenticationType.  Apart from name, config and authenticationType we are not sending anything from server.